basic/exercise4/script4.js

```diff
@@ -1 +1,9 @@
 'use strict';
+
+function leapYear(year) {
+    return ((year % 4 == 0) && (year % 100 != 0)) || (year % 400 == 0);
```

Comment:
Please use === instead of ==
http://stackoverflow.com/questions/359494/which-equals-operator-vs-should-be-used-in-javascript-comparisons
http://stackoverflow.com/questions/523643/difference-between-and-in-javascript